modules/vpc/main.tf

```diff
@@ -0,0 +1,199 @@
+
+
+#Figure out which availability zones are in the region.
+data "aws_availability_zones" "available_azs" {
+  state = "available"
+  filter {
+    name = "opt-in-status"
+    # Currently, no support for Local Zones, Wavelength, or Outpost
+    values = ["opt-in-not-required"]
+  }
+}
+
+#
+# VPC
+#
+resource "aws_vpc" "aap_infrastructure_vpc" {
+    cidr_block = var.infrastructure_vpc_cidr
+    enable_dns_support   = true
+    enable_dns_hostnames = true
+    tags = merge( 
+      {
+        Name = "aap-infrastructure-${var.deployment_id}-vpc"
+      },
+      var.persistent_tags
+    )
+}
+
+#
+# Subnets
+#
+resource "aws_subnet" "aap_infrastructure_subnets" {
+  count = length(var.infrastructure_vpc_subnets)
+  vpc_id                  = aws_vpc.aap_infrastructure_vpc.id
```

Comment:
The spacing style is mixed. Let's pick one way to be consistent. 1 space before and after = or aligned.